lib/models/notification.ts

```diff
@@ -100,11 +135,7 @@ export const RetrievedNotification = tag<IRetrievedNotificationTag>()(
 export type RetrievedNotification = t.TypeOf<typeof RetrievedNotification>;
 
 function toBaseType(o: RetrievedNotification): Notification {
-  return {
-    emailNotification: o.emailNotification,
-    fiscalCode: o.fiscalCode,
-    messageId: o.messageId
-  };
+  return pick(["fiscalCode", "messageId", "channels"], o);
```

Comment:
nop, you lose type safety here

Reply:
the method is still type safe btw. you cannot pick("[unknowProp"]) and you cannot omit required props (the returned object *must* be of type Notification). these are the same constraints you get whn using direct properties assignments.